src/linkedList.js

```diff
@@ -0,0 +1,161 @@
+export default class LinkedList {
+  constructor(){
+    this.num = 0
+    this.head = null
+  }
+
+  Node(data) {
+    this.data = data
+    this.next = null
+  }
+
+  getHeadNode(){
+    return this.head
+  }
+
+  getTailNode(){
+    let currentNode = this.head
+    while(currentNode.next){
+      currentNode = currentNode.next
+    }
+    return currentNode
+  }
+
+    contains(element){
+      if(this.head === null){
+        return false
+      }
+      let currentNode = this.head
+
+      while (currentNode){
+        if(currentNode.data === element){
+          return true
+        }
+        currentNode = currentNode.next
+    }
+    return false
+  }
+
+  find(element){
+    if(this.head === null){
+      return -1
+    }
+    let currentNode = this.head
+    while (currentNode){
+
+      if(currentNode.data === element){
+        return currentNode
+      }
+
+      currentNode = currentNode.next
+
+    }
+    return -1
+  }
+
+  insert(element) {
+    let node = new this.Node(element)
```

Comment:
You shouldn't have to call this on a Node class.
I don't think defining a class inside of another class is allowed in many other programming languages so I would move the Node class to another file and import it into this file.